frame/test_drag_and_drop.py

```diff
@@ -0,0 +1,46 @@
+from display_server import DisplayServer
+from pathlib import Path
+import pytest
+import os
+import re
+import asyncio
+import apps
+
+try:
+    from virtual_pointer import VirtualPointer, Button
+except ModuleNotFoundError as e:
+    pass
```

Comment:
These missing shouldn't be glossed over, should it?

Reply:
Nevermind, I see you took this from the other test. I would probably `import` where it's used instead.